datatune/core/filter.py

```diff
@@ -63,12 +64,12 @@ def llm_inference(
     return df
 
 
-def parse_filter_output(output: str, err: bool = False) -> Optional[bool]:
+def parse_filter_output(output: str | Exception, err: bool = False) -> Optional[bool]:
```

Comment:
support older python versions too by changing this to:

`def parse_filter_output(output: Union[str, Exception], err: bool = False) -> Optional[bool]:`